src/pages/UnsubscribeEmailPage/UnsubscribeEmailPage.tsx

```diff
@@ -54,19 +56,33 @@ export const UnsubscribeEmailPage = () => {
                   <Button
                     color="cta"
                     onClick={() => {
-                      navigate(coLinksPaths.home, {
-                        replace: true,
-                      });
+                      if (isCoLinks) {
+                        navigate(coLinksPaths.home, {
+                          replace: true,
+                        });
+                      } else {
+                        navigate(givePaths.home, {
```

Comment:
does this work if we're on colinks.coordinape.com or app.coordinape.com? can we navigate to the other side of the app?

Reply:
I tested it and it didn't work. it just appends the path to the current domain